test/WebSites/SimpleWebSite/Controllers/HomeController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Mvc;
+
+namespace SimpleWebSite.Controllers
+{
+    public class HomeController
+    {
+        public string Index()
+        {
+            return "Hi from MVC";
+        }
+
+        public string GetUser(int id)
+        {
+            return $"User: {id}";
+        }
+
+        public IDictionary<string, string> Dict()
```

Comment:
Because #3528 asked for something serialized that the functional test can check. The two above it can safely go though.